components/UI/wallets.tsx

```diff
@@ -68,6 +68,29 @@ const Wallets: FunctionComponent<WalletsProps> = ({
                 </Button>
               </div>
             );
+          } else {
+            if (connector.id === "braavos" || connector.id === "argentX") {
+              return (
+                <div className="mt-5 flex justify-center" key={connector.id}>
+                  <Button
+                    onClick={() =>
+                      window.open(
+                        `${
+                          connector.id === "braavos"
+                            ? "https://braavos.app/download-braavos-wallet/"
+                            : "https://chrome.google.com/webstore/detail/argent-x/dlcobpjiigpikoobohmabehhmhfoodbb"
+                        }`
+                      )
+                    }
+                  >
+                    <div className="flex justify-center items-center">
+                      <WalletIcons id={connector.id} />
+                      Install {connector.id}
+                    </div>
+                  </Button>
+                </div>
+              );
+            }
           }
         })}
```

Comment:
Can you also make the wallet modal to pop-up directly on load ? as we discussed today :)